typescript/elasticbeanstalk/elasticbeanstalk-bg-pipeline/index.ts

```diff
@@ -19,8 +19,8 @@ export class CdkStack extends cdk.Stack {
     const bucket = new s3.Bucket(this, 'BlueGreenBucket');
 
     const handler = new lambda.Function(this, 'BlueGreenLambda', {
-      runtime: lambda.Runtime.Python36,
-      code: lambda.Code.directory('resources'),
+      runtime: lambda.Runtime.PYTHON_3_6,
+      code: lambda.AssetCode.asset('resources'),
```

Comment:
Should be:
``` typescript
code: lambda.Code.asset('resources'),
```

Reply:
07c71f42e911be8e002a90ec8c5e7c08ff6f7b6e